oneview_redfish_toolkit/blueprints/resource_block.py

```diff
@@ -149,3 +138,18 @@ def get_resource_block_computer_system(uuid, serial):
         # In case of error log exception and abort
         logging.exception('Unexpected error: {}'.format(e))
         abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+def find_drive_resource(uuid):
+    resource_index = None
+    try:
+        resource_index = g.oneview_client.index_resources \
+            .get('/rest/drives/{}'.format(uuid))
+    except HPOneViewException as e:
```

Comment:
Is there any reason to not let the caller function (`get_resource_block`) treat the HPOneViewException?

Reply:
Because I need get `drive` by `uuid` again in another place, so for this reason I put inside a separeted method.